src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -108,19 +108,30 @@
     <None Remove="Views\Menu\TrustedPathView.xaml" />
     <None Remove="Views\GuidedTour\HtmlPages\Resources\ConnectTheNode.gif" />
   </ItemGroup>
-    <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0-windows' ">
-        <PackageReference Include="System.Configuration.ConfigurationManager" Version="5.0.0" /> 
-    </ItemGroup>
-    <ItemGroup Condition=" '$(TargetFramework)' == 'net48' "> 
-        <Reference Include="System.Configuration" />
-        <Reference Include="PresentationCore" />
-
-    </ItemGroup>
-    <ItemGroup> 
-  <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="2.18.0.1400" />
-  <PackageReference Include="AvalonEdit" Version="4.3.1.9430" CopyXML="true" />
-  <PackageReference Include="Greg" Version="2.5.0.5076" />
-  <PackageReference Include="Microsoft.Web.WebView2" Version="1.0.1264.42" />
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0-windows' ">
+    <PackageReference Include="System.Configuration.ConfigurationManager" Version="5.0.0" /> 
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net48' "> 
+    <Reference Include="System.Configuration" />
+    <Reference Include="PresentationCore" />
+    <Reference Include="PresentationFramework" />
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Drawing" />
+    <Reference Include="System.Runtime.Serialization" />
+    <Reference Include="System.Web" />
+    <Reference Include="System.Windows.Forms" />
+    <Reference Include="System.Xaml" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Xml" />
+    <Reference Include="WindowsBase" />
```

Comment:
These are all implicitly referenced in net6.0 or some such thing?

Reply:
They come automatically when targeting net6-windows + UseWPF
Projects will get a reference to the Microsoft.WindowsDesktop.App package.

You can see all the assembly references under ref/net6 subfolder here:
https://nuget.info/packages/Microsoft.WindowsDesktop.App.Ref/6.0.14